### PR TITLE
Confirmed support for Django 4.0.

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -14,6 +14,7 @@ deps =
     django22: Django>=2.2,<3.0
     django31: Django>=3.1,<3.2
     django32: Django>=3.2,<4.0
+    django32: Django>=4.0,<4.1
```
Comment (felixxm) [inline tox.ini:17]: Typo:
```suggestion
    django40: Django>=4.0,<4.1
```

Comment (hramezani) [reply tox.ini]: Thanks @felixxm :+1:

File: docs/changelog.rst [hunk 2]
```diff
@@ -10,6 +10,8 @@ This page details the changes in the various ``django-formtools`` releases.
 
 - Added Python 3.10 to test matrix.
 
+- Added support for Django 4.0.
```
Comment (felixxm) [inline docs/changelog.rst:13]: I would say "Confirmed", we didn't fix anything here:
```suggestion
- Confirmed support for Django 4.0.
```

**Review by felixxm — APPROVED** [review]
@hramezani Thanks :+1:

File: docs/changelog.rst (hunk 2 above)
Comment (hramezani) [reply docs/changelog.rst]: Yes, sounds better. Thanks again for the suggestion!

**Pull request merged**